lib/releaselib.sh

```diff
@@ -833,3 +846,51 @@ release::docker::release () {
     logrun $GCLOUD config set account $USER@google.com
   fi
 }
+
+###############################################################################
+# Determine whether the release was built with Bazel.
+# @param kube_root - Root of kubernetes tree
+# @return 0 if built with Bazel, 1 otherwise
+release::was_built_with_bazel() {
```

Comment:
the idea behind doing this was that a developer might have a bazel-bin directory from some time ago, but most recently built a release with `make quick-release` - in such a case, we should use the docker-built binaries, not the bazel ones.